### ISSUE TITLE
ATG1/ULK1 kinase complex assembly (GO:1904745) and corresponding regulation terms

### ISSUE THREAD

**Issue by marcfeuermann** [geneontology/go-ontology#12962]
@ukemi @RLovering @Pauldenny @paolaroncaglia

GO:1904745 "ATG1/ULK1 kinase complex assembly" has no annotations yet. PMID:26166702 states that yeast ATG11 might serve as scaffold for the assembly of the complex, but it is considered as part of the complex in mammals and plants (PMID:24563201).  Usually, it is admitted that the complex self-assembles through interaction between ATG13 and ATG17, members of its  2 sub-complexes, respectively (PMID:25998454, 25700739, 25139988).
Regulation terms do not exist yet. I would create them, since the assembly of the complex is the essential step for in initiation of autophagosome formation and target for many signaling pathways.
It marks the frontier between "regulation of autophagy" and "autophagy".
Many annotations could be associated to ATG1/ULK1 kinase complex assembly regulations terms. As an example, the yeast the Ccl1-Kin28 kinase complex positively regulates the ATG1/ULK1 kinase complex assembly (PMID:26567215).
I did not find a negative regulator yet, but we should find some too.
Marc.

**Comment by paolaroncaglia** [comment]
Makes sense to me. @marcfeuermann do you have a TermGenie account to submit the new regulation terms? @ukemi is on TG commit duty this week, so he could approve them. Terms can be used for annotations as soon as a GO ID is created. (Please note that TG no longer sends emails when terms are committed.) Thanks!

**Comment by RLovering** [comment]
What about annotating all members of the complex to this term? The complex requires the components to assemble. I think this might be how biocurators apply annotate assembly terms sometimes? 

eg collagen is associated with http://www.ebi.ac.uk/QuickGO/GAnnotation?a=&goid=GO%3A0030199&termUse=ancestor&relType=IPO%3D&customRelType=IPOR%2B-%3F%3D&evidence=IMP%2CIGI%2CIPI%2CIDA%2CIEP%2CEXP&protein=&tax=&qualifier=&ref=&with=&source=&q=&col=proteinDB%2CproteinID%2CproteinSymbol%2Cqualifier%2CgoID%2CgoName%2Caspect%2Cevidence%2Cref%2Cwith%2CproteinTaxon%2Cdate%2Cfrom%2Csplice&select=normal 
Although not all collagens so perhaps these are specific collagens that have an organizational role.

Ruth

**Comment by marcfeuermann** [comment]
I thought the rule was not to annotate complex members to complex assembly terms ! These proteins have functions as kinase or kinase regulators, but their intrinsic function is not the assembly of the complex. Anyway, the discussion is open.
I have never used TermGenie, but I can try, or ask a colleague who has an account.
Marc.

**Comment by paolaroncaglia** [comment]
Ah, I’ll miss GO… :-)
I’m not entirely sure that the GOC has ever reached a resolution on this. My very personal take is that

a) components of a protein complex should be annotated to the protein complex term

b) gene products that are not part of the protein complex, but aid towards formation of the complex (e.g. by bringing the components together), or prevent its formation, should be annotated to regulation of that protein complex assembly and children

c) the protein complex requires the components to assemble: yes it does in the broad sense that if the components weren’t there, the complex wouldn’t assemble, but _according to GO_, it only does if the components _actively do something to facilitate the assembly_. Simple binding doesn’t count. (And if the components actively do something to facilitate the assembly, to me that’s closer to positive regulation of assembly than assembly itself, but I know that’s a thin line.)

The discussion could go on for days. But I think we have a common agreement in that Ruth notes “Although not all collagens are associated with this term so perhaps these annotations are only applied to the specific collagens that have an organizational role”. So I’d suggest **not to annotate members of the ATG1/ULK1 kinase complex to the assembly term unless we know that they have a special role**.
And yes, this could imply that some annotations to complex assembly terms in general are not (or no longer) fully correct. The MF refactoring work might involve looking at these issues. But for now, I’d rather be cautious in making new annotations that I’m not sure would be fully meaningful.
Please bear with me! :-) (And I see that Marc feels similarly...)

**Comment by paolaroncaglia** [comment]
P.S. @marcfeuermann , sorry to side-track. I can create regulation terms for ATG1/ULK1 kinase complex assembly for you, I'll use PMID:26567215 as a dbxref (and GOC:autophagy), but let me know if you have already seen to this.

**Comment by marcfeuermann** [comment]
@paolaroncaglia
I did not do it yet, just tryed to see how to open TermGenie, so please, feel free to create the terms. You might do it faster and better than me :-)
Thanks a lot
Marc.

**Comment by paolaroncaglia** [comment]
Here they are, ready to use:
ID: GO:1905864 Label: regulation of ATG1/ULK1 kinase complex assembly
ID: GO:1905865 Label: negative regulation of ATG1/ULK1 kinase complex assembly
ID: GO:1905866 Label: positive regulation of ATG1/ULK1 kinase complex assembly

**Comment by paolaroncaglia** [comment]
Wrapping up, @marcfeuermann wrote
"Many annotations could be associated to ATG1/ULK1 kinase complex assembly regulations terms. As an example, the yeast the Ccl1-Kin28 kinase complex positively regulates the ATG1/ULK1 kinase complex assembly (PMID:26567215).
I did not find a negative regulator yet, but we should find some too."
So now Marc could create the annotation as above, and possibly more to the same term and/or to the negative regulation if we find any evidence. The ontology side of this ticket is done. I added a link to this ticket on the last meeting agenda page, next to the relative action item, so it's recorded. Closing now, thanks.
